docs: refer to the web app as "W&B" instead of "W&B App" (#2737)
wandb/docs: models/artifacts/ttl.mdx

Schedule when artifacts are deleted from W&B with a W&B Artifact time-to-live (TTL) policy. When you delete an artifact, W&B marks that artifact as a *soft-delete*. In other words, the artifact is marked for deletion but files are not immediately deleted from storage. For more information on how W&B deletes artifacts, see the [Delete artifacts](./delete-artifacts) page.

> Watch a [Managing data retention with Artifacts TTL](https://www.youtube.com/watch?v=hQ9J6BoVmnc) video tutorial to learn how to manage data retention with Artifacts TTL in W&B.

Reviewer: ```suggestion
Watch the video tutorial [Managing data retention with Artifacts TTL](https://www.youtube.com/watch?v=hQ9J6BoVmnc) video tutorial for details.
```
Similar patterns may exist elsewhere

<Note>
W&B deactivates the option to set a TTL policy for artifacts linked to the Registry. This is to help ensure that linked artifacts do not accidentally expire if used in production workflows.
</Note>
<Note>
* Only team admins can view a [team's settings](/platform/app/settings-page/teams) and access team level TTL settings such as (1) permitting who can set or edit a TTL policy or (2) setting a team default TTL.  
* If you do not see the option to set or edit a TTL policy in an artifact's details in W&B or if setting a TTL programmatically does not successfully change an artifact's TTL property, your team admin has not given you permissions to do so.

> </Note>

Reviewer: Consider rewriting to merge these notes or demoting one.

The preceding code snippet sets the TTL policy for the artifact to 30 days. In other words, W&B deletes the artifact after 30 days.

### Set or edit a TTL policy after you create an artifact

> Use W&B or the W&B Python SDK to define a TTL policy for an artifact that already exists.

Reviewer: We may want to handle patterns like "W&B or the W&B Python SDK" differently in general, maybe something like "the W&B UI or SDK". WDYT?

```

## Deactivate a TTL policy

> Use the W&B Python SDK or W&B to deactivate a TTL policy for a specific artifact version.

Reviewer: Ditto this pattern
